ci/ctest_to_gitlab.sh

```diff
@@ -11,17 +11,30 @@
 #
 
 IMAGE="$1"
-THREADS_PER_NODE="$2"
-SLURM_CONSTRAINT="$3"
+THREADS_MAX_PER_TASK="$2"
+THREADS_PER_NODE="$3"
+SLURM_CONSTRAINT="$4"
+RUNNER="$5"
+
+STAGES="
+  - test
+"
+TIMELIMIT="15:00"
+ARTIFACTS="
+  artifacts:
+    when: always
+    patsh:
```

Comment:
typo?

Reply:
```suggestion
    paths:
```
Definitely, well spotted!